docs/searching.md

```diff
@@ -31,103 +31,214 @@ be searched. In most cases the field value type will be 'Full Text', others may
 ## Per field
 
 ```csharp
-var searcher = myIndex.GetSearcher(); // Get a searcher
+var searcher = myIndex.Searcher; // Get a searcher
 var results = searcher.CreateQuery() // Create a query
  .Field("Address", "Hills") // Look for any "Hills" addresses
  .Execute(); // Execute the search
 ```
 
+### Terms and Phrases
+
+When searching on fields like in the example above you might want to search on more than one word/term. In Examine this can be done by simply adding more terms to the field filter.
+
+```csharp
+var searcher = myIndex.Searcher;
+var results = searcher.CreateQuery()
+  // Look for any addresses that has "Hills" or "Rockyroad" or "Hollywood"
+ .Field("Address", "Hills Rockyroad Hollywood")
+ .Execute();
+```
+
+This means that the `Address` in this example has to match any the values set in the statement. This is because Examine will create a Lucene query like this one where every word is matched separately: `Address:hills Address:rockyroad Address:hollywood`.
```

Comment:
It might be worth adding that this depends on the Analyzer being used. Could be something like: The way that terms are split depends on the Analyzer being used. The StandardAnalyzer is the default. An example of how Analyzers work are:

* StandardAnalyzer - will split a string based on whitespace and 'stop words' (i.e. common words that are not normally searched on like "and")
* WhitespaceAnalyzer - will split a string based only on whitespace
* KeywordAnalyzer - will not split a string and will treat the single string as one term - this means that searching will be done on an exact match

There are many [Analyzers](https://lucenenet.apache.org/docs/4.8.0-beta00016/api/core/Lucene.Net.Analysis.html) and you can even create your own.

Not sure this info exists in this section but an Analyzer will directly affect the term splitting

Reply:
I've now added the text above and written it into the existing example. I've added a link to how to configure analyzers that just lead to https://shazwazza.github.io/Examine/configuration the `Example - Phone Number` section was the only place I could find something about configuring analyzers.